starknet_py/hash/transaction.py

```diff
@@ -30,7 +30,7 @@ def compute_transaction_hash(
     calldata: Sequence[int],
     max_fee: int,
     chain_id: int,
-    additional_data: Sequence[int],
+    additional_data: Optional[Sequence[int]] = None,
```

Comment:
Why is it optional now?

Reply:
Just for the ease of use. If it is not passed, an empty list will be assigned (see below).